test/performance/scale_revision_by_load_test.go

```diff
@@ -141,8 +142,12 @@ func scaleRevisionByLoad(t *testing.T, numClients int) []junit.TestCase {
 						timestamp: time.Now(),
 					}
 					select {
-					case scaleCh <- event:
+					case <-stopCh:
```

Comment:
This doesn't really buy us anything I believe. Should we just ditch the entire `select` block here?